gene/etl/base.py

```diff
@@ -25,7 +25,7 @@ class Base(ABC):
 
     def __init__(self, database: Database, host: str, data_dir: str,
                  src_data_dir: Path,
-                 seqrepo_dir=PROJECT_ROOT / 'data' / 'seqrepo' / 'latest',
+                 seqrepo_dir=APP_ROOT / 'data' / 'seqrepo' / 'latest',
                  *args, **kwargs) -> None:
```

Comment:
Can we also drop args/kwargs here since they aren't used for setting any of the attributes?

Reply:
Sure. I'll add this to a clean up issue I want to make.